cli/daemon/dash/dashapp/src/components/snippets/snippetData.tsx

```diff
@@ -0,0 +1,683 @@
+import {
+  ChatBubbleLeftRightIcon,
+  CircleStackIcon,
+  ClockIcon,
+  CodeBracketIcon,
+  EnvelopeIcon,
+  KeyIcon,
+  Square3Stack3DIcon,
+} from "@heroicons/react/24/solid";
+import React from "react";
+import Code from "~c/snippets/Code";
+
+export interface SnippetSection {
+  slug: string;
+  heading: string;
+  description?: JSX.Element | string;
+  icon: typeof CircleStackIcon;
+  subSections: {
+    heading: string;
+    content: JSX.Element;
+  }[];
+}
+
+const extLink = (href: string, content: string | JSX.Element) => {
+  return (
+    <a href={href} target="_blank" className="link-brandient">
+      {content}
+    </a>
+  );
+};
+
+const docLink = (path: string) => {
+  return <span>{extLink(`https://encore.dev/docs${path}`, "Read the docs")} to learn more.</span>;
+};
+
+const apiSection: SnippetSection = {
```

Comment:
This would be great if we actually built it / generated it from the docs within this repo so we don't now need to keep these and the markdown file in sync